portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java

```diff
@@ -206,6 +206,31 @@ public LiferayPortletURL create(
 
 		return _create(portletRequest, portletId, layout, lifecycle, null);
 	}
+	/*
+	Tried to minimalizing overwriting existing code as much as possible.
+	 */
+	@Override
+	public LiferayPortletURL create(
+		HttpServletRequest httpServletRequest, String portletId,
+		String lifecycle, boolean isItFromNotification) {
```

Comment:
Hi @kristofloneck could you please rename `isItFromNotification` parameter to `notification`?